ci/ray_ci/container.py

```diff
@@ -93,6 +93,7 @@ def _get_run_command(
             "NVIDIA_DISABLE_REQUIRE=1",
             "--volume",
             "/tmp/artifacts:/artifact-mount",
+            "--privileged",
```

Comment:
please make this a flag so that it is enabled by default for everything.

least privilege principle.

Reply:
the flag can be passed down through this script https://github.com/ray-project/ray/blob/master/ci/ray_ci/tester.py#L40; poke me if you need any help ;)